src/ripple/core/JobTypes.h

```diff
@@ -138,14 +138,15 @@ add(    jtNS_WRITE,      "WriteNode",               0,        true,  0ms,     0m
     {
         assert (m_map.find (jt) == m_map.end ());
 
-        std::pair<Map::iterator,bool> result (m_map.emplace (
+        auto const [_, inserted] = m_map.emplace (
```

Comment:
In this case, I would probably use `[[maybe_unused]]` on the whole structured binding declaration rather than two `(void)` statements below.

Reply:
I'm going to get rid of the `(void)` statements all together. I'll add them back (or add `[[maybe_unused]]` if compilers start warning about them.

Reply:
Fixed in 71c328bbc `* [fold] Remove "maybe unused" warning work around:`